common/scripts/lint_go.sh

```diff
@@ -19,9 +21,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-updatecommon:
-	@git clone https://github.com/istio/common-files
-	@cd common-files
-	@git rev-parse HEAD >.commonfiles.sha
-	@cp -r common-files/files/* common-files/files/.[^.]* .
-	@rm -fr common-files
+golangci-lint run -j 8 -c ./common/config/.golangci.yml
```

Comment:
this may fail with OOM. We had GOGC=25 set before which makes the GC more aggressive

Reply:
Possible. I'll figure that out specifically in the next step when I start to leverage the new build logic, including the standardized linter suites.

Reply:
oh right this isn't used yet. in my other pr I tried to get rid of it and got oom immediately which is why I was surprised this didnt